webapp/public/index.html

```diff
@@ -87,7 +86,9 @@
             "vs/language/typescript/src/worker": "/blb/vs/language/typescript/src/worker.js",
             "lzma/lzma_worker-min.js": "/blb/lzma/lzma_worker-min.js",
             "smoothie/smoothie_compressed.js": "/blb/smoothie/smoothie_compressed.js",
-            "pxtblockly.js": "/blb/pxtblockly.js"
+            "pxtblockly.js": "/blb/pxtblockly.js",
+            "blockly.css": "/blb/blockly.css",
+            "rtlblockly.css": "/blb/rtlblockly.css"
```

Comment:
so we load both?

Reply:
we load the one we need (we used to load both)

Reply:
Cool!